src/ui/state/manager.rs

```diff
@@ -211,7 +233,7 @@ impl State for ManagerHomeState {
             Command::perform(get_blockheight(self.revaultd.clone()), Message::BlockHeight),
             Command::perform(list_vaults(self.revaultd.clone(), None), Message::Vaults),
             Command::perform(
-                list_spend_txs(self.revaultd.clone()),
+                list_spend_txs(self.revaultd.clone(), Some(&["non_final"])),
```

Comment:
I prefer we create an enum `revaultd::model::SpendTxStatus`  with the different statuses
check `revaultd::model::VaultStatus` and `list_vaults` for inspiration.